packages/pagination/src/PaginationContent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Util } from 'reactstrap';
+import BlockUI from 'react-block-ui';
+import { usePagination } from './Pagination';
+
+const PaginationContent = ({
+  component: Component,
+  loadingMessage,
+  itemKey,
+  loader = false,
```

Comment:
Why not `defaultProps`?

Reply:
I was torn between which one to use, as inlining them with the prop args sometimes is easier to read then going to defaultProps declaration and matching the key value to the prop arg. I will refactor it to just use the defaultProps instead.